target/linux/ramips/dts/mt7621_asus_rt-ax1800hp.dts

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "asus,rt-ax1800hp", "mediatek,mt7621-soc";
+	model = "ASUS RT-AX1800HP";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+		bootargs-override = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "blue:power";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_POWER;
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		key-restart {
+			label = "reset";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		key-wps {
+			label = "wps";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x80000>;
+			read-only;
+		};
+
+		/*
+		 * u-boot gets split here while keeping u-boot read-only,
+		 * which allows safe usage of fw_setenv
+		 */
+		partition@80000 {
+			label = "u-boot-env";
+			reg = <0x80000 0x60000>;
+		};
+
+		partition@e0000 {
+			label = "nvram";
+			reg = <0xe0000 0x100000>;
+			read-only;
+		};
+
+		factory: partition@1e0000 {
+			label = "factory";
+			reg = <0x1e0000 0x100000>;
+			read-only;
+
+			compatible = "nvmem-cells";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			macaddr_factory_4: macaddr@4 {
+				reg = <0x4 0x6>;
+			};
+		};
+
+		factory2: partition@2e0000 {
+			label = "factory2";
+			reg = <0x2e0000 0x100000>;
+			read-only;
+		};
+
+		partition@3e0000 {
+			label = "kernel";
+			reg = <0x3e0000 0x400000>;
+		};
+
+		partition@7e0000 {
+			label = "ubi";
+			reg = <0x7e0000 0x2e00000>;
+		};
+
+		partition@35e0000 {
+			label = "firmware2";
+			reg = <0x35e0000 0x3200000>;
+		};
+
+		/* Last 8M possibly store the bad block table */
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie1 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0000 0 0 0 0>;
+		mediatek,mtd-eeprom = <&factory 0x0000>;
+		mediatek,disable-radar-background;
+	};
+};
+
+&gmac0 {
+	nvmem-cells = <&macaddr_factory_4>;
+	nvmem-cell-names = "mac-address";
+};
+
+&gmac1 {
+	status = "okay";
+	label = "lan4";
```

Comment:
`lan4` on gmac1 ?

Reply:
yeah wait why it 's lan4 https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_netgear_wac104.dts is also on gmac1?

Reply:
Also what it's lan3 https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_ampedwireless_ally-r1900k.dts is also on gmac1?

Reply:
First device mentioned by you only has lan ports labeled since it's an access point, and maybe it was decided to make one of those lan ports a wan port, for proper routing speeds. The 2nd one I don't know what's going on there, probably a mistake.
But what is the reason on AX1800HP to put lan4 on the 2nd gmac ?

Reply:
> But what is the reason on AX1800HP to put lan4 on the 2nd gmac ?

well the lan4 is conected to gmac2

Reply:
Well gmac 2 is config to the switch port 0 (not sure how to change it ) and  the port 0 is lan4.

Reply:
Well gmac 2 is config to the switch port 0 (not sure how to change it ) and the port 0 is lan4 since there r other rotuer r configed smailer so that is not a issue i think.

Reply:
era... maybe it is a issue . Let me fix it then

Reply:
it seem to didnt want to be  compiled after the fix.

Reply:
Hello, I have installed openwrt ax5u in ax1800hp but I don't know how to use it
How do I go back to the asus interface?
I used: Firmware Restoration failed the wireless router is not in rescue mode
Hope the helping

Reply:
@Znevna  i fixed this and i have tested this it work.

Reply:
have you tried pppoe on it?

Reply:
I tried recording pppoe from ax1800hp via openwrt but it doesn't work

Reply:
> did i tell u to not ask in here, bitch ?


![image](https://user-images.githubusercontent.com/70055899/216352756-8d3261f2-716c-4ccb-a241-242564bcb7b8.png)